md4/src/lib.rs

```diff
@@ -38,30 +38,30 @@ pub struct Md4 {
 
 impl Md4State {
     fn process_block(&mut self, input: &Block) {
-        fn f(x: u32, y: u32, z: u32) -> u32 {
+        fn foo(x: u32, y: u32, z: u32) -> u32 {
```

Comment:
`f`, `g` and `h` are names used in the description of the algorithm, so it's much better to use them instead of `foobar`s.

Reply:
No problem.